### ISSUE TITLE
📝 Documentation: how to release beta versions of a package?

### ISSUE THREAD

**Issue by johnnyreilly** [JoshuaKGoldberg/create-typescript-app#912]
### Bug Report Checklist

- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/create-typescript-app/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

so way back when, I used create-typescript-app under its original name to create https://github.com/johnnyreilly/rehype-cloudinary-docusaurus

docusaurus v3 is in beta and it's all about the ESM. To migrate my plugin I need to take it from commonjs to esm and  I bet I get it wrong along the way

Question: does create-typescript-app support releasing beta versions? I don't spot anything in the FAQs

https://github.com/JoshuaKGoldberg/create-typescript-app/blob/main/docs/FAQs.md

**Comment by johnnyreilly** [comment]
From conversation with @JoshuaKGoldberg :+1: 

> I can tell you now that I personally don't know how to release versions with any particular tag automatically on npm. You can put whatever version string you want in package.json and different tools -`npm publish`, `release-it`, etc.- may support them. But I haven't looked into it. Please do file that feature request 😄 I'd love to see it
> This has come up a few times before and my strategy was to manually locally remove the provenence field temporarily from package.json, then publish manually. not great.

**Comment by JoshuaKGoldberg** [comment]
Yeah I'm not sure whether this is a _documentation_ or _feature_ request. But accepting PRs either way!

* If it _is_ currently supported, let's add a docs FAQ entry at the least
* If it _isn't_ currently supported, let's look into adding a feature... and maybe also a docs FAQ entry

**Comment by JoshuaKGoldberg** [comment]
@allcontributors add @johnnyreilly for idea
